RelhaxModpack/RelhaxModpack/Utils/Translations.cs

```diff
@@ -5041,7 +5042,7 @@ public static void LoadTranslations()
             English.Add("GcDownloadStep2TabDescription", "Close the game center (application will detect closure)");
             German.Add("GcDownloadStep2TabDescription", "Schließe das Spiel (das Programm erkennt das Schließen)");
             Polish.Add("GcDownloadStep2TabDescription", "Proszę zamknąć WG Game Center (aplikacja automatycznie wykryje zamknięcie).");
-            French.Add("GcDownloadStep2TabDescription", TranslationNeeded);
+            French.Add("GcDownloadStep2TabDescription", "Fermer le game center (L'application va détecter la fermeture");
```

Comment:
potential missed closing parenthesis? @Toshiro-Relhax

Reply:
My humble opinion, just fix it for him ;) He dedicated his time to finish the tranlsation, which was severely outdated. It's cruel to hold it based on a single ``)`` :D

Reply:
bruh.

i'm not holding him on a single `)`. I will gladly do it for him. If he gets to it first or I tomorrow morning, who ever :)

Reply:
Haha, X-mas challenge! :D